index.html

```diff
@@ -6,21 +6,24 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="./assets/style/style.css">
     <script src="assets/js/main.js" defer></script>
 </head>
 
 <body>
-    <section>
-        <center>
+    <section id="listOfBooksSection">
+<h1>Awesome Books</h1>
+        <div class="booksContainer">
```

Comment:
<img width="1020" alt="1" src="https://github.com/DjibrilM/Awesome-books/assets/77843050/a4ac6c9c-41b8-4a2f-9b7c-3059f576c792">

- [ ] Dears, @DjibrilM and @AsumVictor I think you still have some steps to complete before the approval, as the image shows, you are required to follow that wireframe for your design, and yes you did but not completely, we are missing the page to the list book, contact and add book, please consider separating each page as it is required in the project.

Your project actually looks like this

<img width="829" alt="2" src="https://github.com/DjibrilM/Awesome-books/assets/77843050/059e1933-1fd1-437d-a893-bc12bcd6274d">

Reply:
- [ ] Please do not take into consideration these comments, the student contacted me about the mistake he did in submitting this PR, and I got the `Awesome books: a complete website with navigation` by mistake from the student... In case you are reviewing this as a second or third review, please approve it.